src/lib/config.js

```diff
@@ -15,6 +15,69 @@ module.exports = {
     BASE_URL: process.env.EXTERNAL_WEBSITE_HOST || "http://localhost:5000",
     PATHS: {
       KBV: "/kbv",
+      OAUTH2_CALLBACK_PATH: "/oauth2/callback",
+      PROVE_IDENTITY_ANOTHER_WAY: "/prove-identity-another-way",
+      DONE: "/done",
+      QUESTION: "/question/",
```

Comment:
I think we should either name this `QUESTION_PREFIX` or move the trailing slash to where the routes are defined